### ISSUE TITLE
NTR: Guanidinoacetic acid in CSF high (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8445]
New term request Guanidinoacetic acid in CSF high (segal_051118194303): GAA; cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000396
- label: Elevated CSF guanidinoacetic acid concentration
- parents: [Abnormal CSF amino acid concentration (HP:0500184)](https://hpo.jax.org/app/browse/term/HP:0500184)
- pmid: PMID:12701824
- def: The concentration of guanidinoacetic acid in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8445
- synonyms:
